### ISSUE TITLE
Option to only show top directories

### ISSUE THREAD

**Issue by BarbzYHOOL** [Kwpolska/upass#23]
When we launch upass, we get to see base directories and all their subdirs, it's very handy but there could be a switch to turn it off and see only the base/root dirs.

upass --base-dir
upass -b

**Comment by thurstylark** [comment]
I second this. I was expecting a folder structure tree much like the output of `pass show`, and I still kind of want that. Would be nice to set this in the config file imo.

**Comment by Kwpolska** [comment]
You are talking about two different features.

@BarbzYHOOL wants less flattening of the structure.

@thurstylark wants to see everything on one page.

I am fine with both being implemented as options. The default view is fine IMO. PRs to add either of the options are always welcome.

**Comment by thurstylark** [comment]
@Kwpolska You're right about what I actually said (and I still kinda think it's a good idea), but I misspoke about my idea.

I was instead envisioning a display that just shows the top-level dirs, and then shows subdirs after selecting a top-level dir. 

Again, I think both would be cool, but I definitely would prefer the latter.
